USING_VANAGON.md

```diff
@@ -0,0 +1,42 @@
+# Using Vanagon
+- Note that these docs are incomplete. Docs for all parts of vanagon are on their way
+
+## Creating a Vanagon Project for Windows
+
+### Overview
+Vanagon projects for windows use [WiX](http://wixtoolset.org/) to create MSIs that will install on native windows platforms.  Vanagon uses the same process between all platforms, including windows, to build and stage artifacts before creating an installer.  Vanagon only uses WiX files for the actual installer itself, so the smaller / simpler your install experience, the less WiX you need. You can find example WiX files under [examples](https://github.com/puppetlabs/vanagon/examples/resources/windows/wix).  These files should provide a good start to create an MSI using vanagon.
```

Comment:
As we discussed earlier, most of the files inside vanagon don't need to be specific to vanagon. They are good generic starting points but there are likely many immediate use cases to change them. So I am going to opt for removing them as "vanagon defaults" and just listing them as starting points under examples. The idea being you could just copy them verbatim in to your project and they will work without changing them, but if you need to change them they already exist in your project.